src/cosalib/builds.py

```diff
@@ -0,0 +1,112 @@
+import os
```

Comment:
This patch should be part of the previous commit I think, where we delete it from `cmdlib.py`.